### ISSUE TITLE
Help us test Codesandbox with Overmind

### ISSUE THREAD

**Issue by christianalfoni** [codesandbox/codesandbox-client#2375]
Codesandbox is about to switch out its state management system from [Cerebral](https://cerebraljs.com) and [mobx-state-tree](https://mobx-state-tree.gitbook.io/docs/) to [Overmind](https://overmindjs.org).

**The main reason for switching it out is:**

1. Get [Typescript](https://typescriptlang.org) support
2. Avoid HOCs in our React components 
3. Keep the state management devtool experience

**As a result Codesandbox will be:**

1. Less error prone as Typescript has our back
2. Cleaner codebase as we are not worried about refactoring
3. More contribution friendly as Overmind has an "API-less" api

**How to help:**

1. Go to [codesandbox.stream](https://codesandbox.stream/s/github/CompuIves/new-sandbox), which is the test environment of Codesandbox. You will enter a React sandbox which you can fork. Currently you can not test other types of sandboxes as they are not part of the staging environment. You will not see any of your production content here either
2. Open up the browser console and run the following exposed global function:

<img width="1440" alt="Screenshot 2019-09-04 at 11 35 07" src="https://user-images.githubusercontent.com/3956929/64244557-913f2280-cf09-11e9-92c4-f2b910361ee1.png">

3. The browser refreshes automatically and you should now see the Overmind logo down left:

<img width="1439" alt="Screenshot 2019-09-04 at 11 46 30" src="https://user-images.githubusercontent.com/3956929/64244607-a9af3d00-cf09-11e9-874e-1472a98c9fba.png">

4. To move back you can do the same with: `useOvermind(false)`

**What to test?**

Basically everything. Just use Codesandbox as you normally do. You will soon be able to do this switch on [codesandbox.io](https://codesandbox.io) as well.

Any issues can be reported here on Github. Please include **Overmind** in the title of the issue.

**Comment by krvajal** [comment]
I just tried to follow the steps and it seems  that
> run the following exposed global function

does not work. `useOvermind` is not defined
